src/backends/onnxruntime.c

```diff
@@ -69,11 +69,17 @@ DLDataType RAI_GetDLDataTypeFromORT(ONNXTensorElementDataType dtype) {
   return (DLDataType){ .bits = 0 };
 }
 
-OrtValue* RAI_OrtValueFromTensor(RAI_Tensor* t) {
+OrtValue* RAI_OrtValueFromTensor(RAI_Tensor* t, RAI_Error *error) {
   // TODO: create outside and pass?
   OrtAllocatorInfo* allocator_info;
   OrtStatus* status;
   status = OrtCreateCpuAllocatorInfo(OrtArenaAllocator, OrtMemTypeDefault, &allocator_info);
+  if (status != NULL) {
```

Comment:
It's possible to handle error with a "goto error" to an "error" label at the end of the function, instead of repeating the error block for every failure.

Reply:
Done, I just had to enclose code where variable length arrays were initialized in braces to avoid the compiler complaining about goto jumping over variable initialization.